README.md

```diff
@@ -81,6 +81,9 @@ git clone https://github.com/catppuccin/tmux.git ~/.config/tmux/plugins/catppucc
 <!-- x-release-please-start-version -->
 
 ```bash
+# Load catppuccin theme
+run ~/.config/tmux/plugins/tmux/catppuccin.tmux
```

Comment:
Couple of things:

1. This example config doesn't use any status line modules that require TPM to be run twice
2. The `run` lines are in the wrong spots